### PR TITLE
[Keysight/Ixia] [Doc] Topology information to execute control pane and data pane tests using Ixia traffic generator against SONiC DUT

### LINKED ISSUE #1887: Topology information to execute control pane and data pane tests using Ixia traffic generator against SONiC  DUT
- [ ] Update of topology diagram SONiC DUT directly connected with Keysight Ixia traffic generator  (IxNetwork)

- [ ] Deployment of IxNetwork API Server

### PR DESCRIPTION
**Pull request by SubhajitPalKeysight**
### Description of PR

- This document gives an overview of the topology where SONiC DUT is directly connected with Keysight’s protocol emulator and traffic generator (IxNetwork)

Summary:
Incorporation of  #1887 

### Type of change

- [ ]  Added Topology Diagram ( ansible/doc/img )

- [ ] Update topology diagram – SONiC DUT directly connected with Keysight Ixia traffic generator ( /ansible/doc/README.testbed.Overview.md )

- [ ] Deployment of Ixia API server docker ( ansible/doc/README.testbed.VsSetup.md )

#### What is the motivation for this PR?

- This will help user to deploy and run tests against SONiC DUT using Ixia traffic generator easily. User can build data pane and control pane tests and run those easily.

#### How did you verify/test it?
- NA Only Doc Change

### CODE REVIEW

File: ansible/doc/README.testbed.Overview.md [hunk 1]
```diff
@@ -54,3 +54,35 @@ DUT front panel port is directly connected to one of PTF container ports. Usuall
...
+
+- Single DUT Topology
+![](img/single-dut-topology.png)
+
+
+- Multiple DUT Topology
+![](img/multiple-dut-topology.png)
+
+
+- Multiple IxNetwork Topology
```
Comment (lolyu) [inline ansible/doc/README.testbed.Overview.md:74]: Could you please explain the difference between `Multiple DUT Topology` and `Multiple IxNetwork Topology`? I cannot tell any difference based on the images.

Comment (SubhajitPalKeysight) [reply ansible/doc/README.testbed.Overview.md:74]: I’ve updated Multiple IxNetwork Topology ( https://github.com/SubhajitPalKeysight/sonic-mgmt/blob/master/ansible/doc/img/multiple-ixnetwork.PNG ).

Multiple DUT Topology : Let’s  consider an Ixia chassis has max N test ports. If a test topology has a SUT (multiple DUTs) and total port requirement is <=N, then single Ixia Chassis (IxNetwork) is sufficient to test such topology containing multiple DUTs. There is no need to have multiple Ixia Chassis.

Multiple IxNetwork Topology : In a test topology which requires > N ports, multiple Ixia Chassis will be needed. Such a system can be managed by a single IxNetwork API Server capable of spawning multiple sessions.

Comment (SubhajitPalKeysight) [reply ansible/doc/README.testbed.Overview.md:74]: would you please review and let us know your comment.

**Comment by lolyu** [comment]
The rest LGTM, can you please clean up and format your commits, like remove your edit history?

**Pull request merged**